### DISCUSSION TITLE
Integration with Souin cache and Caddy

### DISCUSSION THREAD

**Discussion by adammakowskidev** [maypok86/otter#94] [Ideas]
Hi
@maypok86
Can you help integrate Otter cache in Souin?
https://github.com/darkweak/souin

Currently, the integration exists but contains errors... Memory leak under heavy load.
As an Otter creator, can you support the creators of Souin?

Regards

**Comment by maypok86** [comment]
I am very sorry that this happened. Can you describe your problem in more detail?

**Comment by adammakowskidev** [comment]
Hi.  When the caddy is heavily loaded, it simply uses a lot of RAM. I don't know the exact problem, I'm not that good at programming.  If you want, I can explain how to install caddy with souin cache and test Otter cache.

**Comment by adammakowskidev** [comment]
@maypok86 Maybe I'll ask it another way, how much RAM does Otter need for 10,000 keys?
